modules/claims_api/config/schemas/2122.json

```diff
@@ -3,17 +3,160 @@
   "description": "Form 2122 Schema",
```

Comment:
Adding in the top level fields
data:
  type:
  attributes

noticed the fixture wasn't working